class-00/lab-b/starter-code/app.js

```diff
@@ -184,17 +207,17 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+// This refers to an instantiation of the Student object.
 // 2. What is "this" when joe.scopeArrow() is invoked?
-//
+// This in the arrow function is the window object
 // 3. Explain why "this" is different when an arrow function is used.
-//
+// This is different because of the bubbling up effect there is no containment because the this does not refer to the method's  inner function. 
```

Comment:
Correct! This does not refer to the object 'this' is being used within and instead 'bubbles up' to the window object.